ovs/extensions/healthcheck/openvstorage/openvstoragecluster_health_check.py

```diff
@@ -518,12 +518,12 @@ def check_model_consistency(logger):
                                                                         vpool_guid=vp.guid,
                                                                         vpool_name=vp.name,
                                                                         node_id=OpenvStorageHealthCheck.MACHINE_ID)
-                voldrv_client = src.LocalStorageRouterClient(config_file)
 
                 # collect data from volumedriver
                 try:
+                    voldrv_client = src.LocalStorageRouterClient(config_file)
                     voldrv_volume_list = voldrv_client.list_volumes()
-                except ClusterNotReachableException:
+                except (ClusterNotReachableException, RuntimeError):
                     logger.failure("Seems like the volumedriver '{0}' is not running.".format(vp.name),
                                    'discrepancies_ovsdb_{0}'.format(vp.name))
```

Comment:
Please add the actual exception message too, e.g.: 
RuntimeError: Failed easy perform [code: 7] Couldn't connect to server